napari/layers/base/base.py

```diff
@@ -103,7 +99,7 @@ class Layer(VisualWrapper, KeymapMixin, ABC):
     Notes
     -----
     Must define the following:
-        * `_get_shape()`: called by `shape` property
+        * `_get_range()`: called by `range` property
```

Comment:
we're now getting the range on each layer - which has min / max instead of just the shape. this was needed as layers like `Points` and `Shapes` need to be described by their range